src/goDeclaration.ts

```diff
@@ -80,7 +82,7 @@ export function adjustWordPosition(document: vscode.TextDocument, position: vsco
 
 const godefImportDefinitionRegex = /^import \(.* ".*"\)$/;
 function definitionLocation_godef(input: GoDefinitionInput, token: vscode.CancellationToken): Promise<GoDefinitionInformation> {
-	let godefTool = input.isMod ? 'godef-gomod' : 'godef';
+	let godefTool = 'godef';
```

Comment:
Let's not change this in this PR. I have a separate issue tracking the need to use `godef` instead of `godef-mod`, now that the changes from the latter have been merged to the former